src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilationWithAnalyzers.cs

```diff
@@ -1227,8 +1228,9 @@ public async Task<AnalyzerTelemetryInfo> GetAnalyzerTelemetryInfoAsync(Diagnosti
             try
             {
                 var actionCounts = await GetAnalyzerActionCountsAsync(analyzer, cancellationToken).ConfigureAwait(false);
+                var suppressionActionCounts = analyzer is DiagnosticSuppressor ? 1 : 0;
```

Comment:
>suppressionActionCounts [](start = 20, length = 23)

Is it never greater than 1?

Reply:
Yes, the suppressor API does not allow registering multiple callbacks, but just overriding a single `ReportSuppressions(SuppressionContext)` API per analyzer.